crates/volta-core/src/tool/node/resolve.rs

```diff
@@ -249,15 +249,11 @@ fn resolve_node_versions(url: &str) -> Fallible<RawNodeIndex> {
             debug!("Node index cache was not found or was invalid");
             let spinner = progress_spinner(&format!("Fetching public registry: {}", url));
 
-            let (_, headers, response) = attohttpc::get(url)
-                .send()
-                .and_then(Response::error_for_status)
-                .with_context(registry_fetch_error("Node", url))?
-                .split();
+            let mut response: reqwest::blocking::Response =
+                reqwest::blocking::get(url).with_context(registry_fetch_error("Node", url))?;
```

Comment:
I think we want to make sure to preserve the `.and_then(Response::error_for_status)` call here, which converts an error status code (4XX or 5XX) into an actual error.